test/ably/restchannelpublish_test.py

```diff
@@ -348,3 +348,13 @@ def test_wildcard_client_id_can_publish_as_others(self):
 
         self.assertEqual(messages[0].client_id, some_client_id)
         self.assertIsNone(messages[1].client_id)
+
+    def test_publish_message_with_connection_key(self):
```

Comment:
Why is this commit part of the "Added X-Ably-Version and X-Ably-Lib HTTP headers" PR?
Also, can you add the test code to the test itself so that a simple search of your code will reveal all tests that match that spec code?